src/styles/header.scss

```
@@ -0,0 +1,89 @@
+@import './variables';
+
+html {
+  font-family: Roboto, Raleway, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-width: 100%;
+  flex-shrink: 0;
+  background: #fff;
+
+  &__logo {
+    display: flex;
+    align-items: center;
+
+    &-img {
+      width: 40px;
+      height: 40px;
+      background: lightgray 50% / cover no-repeat;
```

minor - but this is redundant:

```suggestion
      background: lightgray 50% ;
```

Cover and no-repeat won't have any effect on img src
If you want to control how the image  (set by src) is positioned - check out `object-position` and `object-fit`  CSS properties. For instance:

```
object-fit: cover;
```